clients/cypress-e2e/cypress/support/constants.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Based off of fides.toml
```

Comment:
Looks like Cypress some [other ways](https://docs.cypress.io/guides/guides/environment-variables#Option-3-CYPRESS_) of sharing environment info with a test runner. I don't know if environment variables would actually be convenient, but it seems likely we'd outgrow copying into this file quickly.

Reply:
hmm okay I'd like to punt on this one until this becomes inconvenient. mostly because I'm not sure how best to use env info in a convenient way yet, but maybe that'll become clear once we arrive at a real problem

Reply:
never mind, I thought through it and realized how to do it now 😄 https://github.com/ethyca/fides/pull/2241/commits/3fd039668b51db8a54323aa09abced0b826a594c

that should satisfy the req
> MUST support targeting different hosts via environment variables, so we can run against staging / demo / etc.